closure/goog/testing/net/xhrio_test.js

```diff
@@ -55,6 +55,44 @@ function testStaticSend() {
   assertEquals('sendInstances_ after READY', 0, sendInstances.length);
 }
 
+function testStaticSendWithFactory() {
+  var xhrFactoryCalled = 0, optionsFactoryCalled = 0, callbackInvoked = 0;
```

Comment:
Please use [5.1.2 One variable per declaration](https://google.github.io/styleguide/jsguide.html#features-one-variable-per-declaration) even though the closure-library seems to conform to the old JavaScript style guide.